tinker/events/events_controller.py

```diff
@@ -436,10 +436,17 @@ def get_search_results(self, selection, title, start, end):
                 if form['title'] == title:
                     toReturn.append(form)
             elif hasDates and checkDates:  # Date range search
-                if (end - eventEnd) >=datetime.timedelta(seconds=0) and (start - eventStart) <=datetime.timedelta(seconds=0):
+                if self.compare_timedeltas(eventEnd, end)== 1 and self.compare_timedeltas(eventStart, start)==2:
                     toReturn.append(form)
         return toReturn, '-'.join(selection) == 'user-events'
 
     def compare_timedeltas(self, a, b):
-        # If a and b are both datetime objects then if b comes after a it will return true
-        return (b-a) > datetime.timedelta(seconds=0)
\ No newline at end of file
+        # If a and b are both datetime objects then if b comes after a it will return 1
+        if(b-a) >= datetime.timedelta(seconds=0):
```

Comment:
datetime.timedelta(seconds=0) could be set as a variable, because this constant is used multiple times